### PR TITLE
Removing PendingStaging field from the files table

### PR DESCRIPTION
**Pull request by cmadjar**
This pull request removes the unused `PendingStaging` field from the `files` table as was discussed during the LORIS imaging meeting of February 2nd, 2018.

See also:
- associated redmine ticket: https://redmine.cbrain.mcgill.ca/issues/13876
- associate PR on the LORIS-MRI side: https://github.com/aces/Loris-MRI/pull/292

### CODE REVIEW

File: SQL/0000-00-00-schema.sql [hunk 1]
```diff
@@ -637,7 +636,6 @@ CREATE TABLE `files` (
   KEY `sessionid` (`SessionID`),
   KEY `outputtype` (`OutputType`),
   KEY `filetype_outputtype` (`FileType`,`OutputType`),
-  KEY `staging_filetype_outputtype` (`PendingStaging`,`FileType`,`OutputType`),
```
Comment (driusan) [inline SQL/0000-00-00-schema.sql]: should there also be a cleanup patch for existing LORIS instances?

Comment (cmadjar) [reply SQL/0000-00-00-schema.sql]: @driusan There definitely should be and I created it on Friday. I simply forgot to commit it to the branch... Good catch! Thanks!! Should be fixed now.

**Comment by driusan** [comment]
@cmadjar Why is this blocked? Does it depend on the Loris-MRI PR?

**Comment by ridz1208** [comment]
@driusan indeed. the PR is linked in the original description

**Comment by cmadjar** [comment]
> @cmadjar Why is this blocked? Does it depend on the Loris-MRI PR?

Yes, it is linked to a LORIS-MRI PR: aces/Loris-MRI#292

**Comment by cmadjar** [comment]
> @cmadjar can it be tested though ?

I am uncertain what represent the **it** in your sentence.

**Comment by ridz1208** [comment]
This PR does not have the "Pass Manual Test" label. Can the PR be tested in its current state ?

**Comment by cmadjar** [comment]
Aah, I see. Yes, it can. To test the PR: 
- make sure the SQL patch runs and removes the PendingStaging column from the files table
- make sure that the imaging browser pages load

Note: if you want to be very diligent, you can check out aces/major and then the imaging browser would crash as it is by default looking at the useless PendingStaging field in the files table to load data. Hence the importance of changing the code along with the patch ;).

Hope that makes sense.

**Comment by driusan** [comment]
@cmadjar conflicts

**Comment by MounaSafiHarab** [comment]
@cmadjar 
needs rebasing

**Comment by cmadjar** [comment]
@colezweber Sorry I dismissed your review by squashing all my commits into one... If you could review it again, that would be awesome!

File: SQL/Archive/20.0/2018-04-05_RemovePendingStagingFromFiles.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-- Remove the PendingStaging field from the files table as discussed during
```
Comment (zaliqarosli) [inline SQL/Archive/20.0/2018-04-05_RemovePendingStagingFromFiles.sql]: this should go to new patches directory

Comment (cmadjar) [reply SQL/Archive/20.0/2018-04-05_RemovePendingStagingFromFiles.sql]: @zaliqarosli Nice catch! I moved the patch to the `Cleanup_patches` directory :). Please re-review.

**Comment by driusan** [comment]
@MounaSafiHarab can you merge this (along with the MRI side..)

**Pull request merged**